frontend/app/model/Eventbrite.scala

```diff
@@ -187,6 +189,10 @@ object Eventbrite {
       case t: InternalTicketing => t
     }
 
+    val limitedAvailabilityText = "Last tickets remaining"
```

Comment:
I think as it's repeated it feels nice to have it in here, I think we do the same for all other status messaging too.